std_msgs/Header header
RealsenseTarget[] realsense_targets
